Makefile

```diff
@@ -45,7 +45,7 @@ vet:
 
 test:
 	# Runs only unit tests
-	go test -race -tags=\!integration $$(go list ./... | grep -v /vendor/ | grep -v /cmd/)
+	go test -race -tags=\!integration\!property $$(go list ./... | grep -v /vendor/ | grep -v /cmd/)
```

Comment:
I'd use property_test or proptest to make this super clear in the test files that don't have _prop in the filename. Integration could be changed as well (but stands alone better than property).